tests/discussions.test.js

```diff
@@ -0,0 +1,364 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const Discussions = require('../src/models/discussions');
+const Accounts = require('../src/models/accounts');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+let accessToken= null;
+let idDiscussion = null;
+
+describe("POST Tests Discussion",() =>{
+    before((done) =>{
+        const login = {
+            username: "MiroStar",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("POST /Discussions", (done) =>{
+        const discussion = {
+            title: "¿Qué opinan de los cuentos en zapoteco?",
+            comment: "Me gustaría saber que opinan de los cuentos que esta y donde puede encontrar más",
+            theme: "doubt",
+            idAccount: "6168cf9563929f8f000c7614"
+        }
+        chai.request(server).post("/discussions").send(discussion)
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(201);
+            response.body.should.be.a("object");
+            response.body.should.have.property("_id");
+            response.body.should.have.property("comment");
+            response.body.should.have.property("title");
+            response.body.should.have.property("theme");
+            response.body.should.have.property("idAccount");
+            response.body.should.have.property("dateCreation");
+            response.body.should.have.property("status");
+            idDiscussion = response.body._id;
+            done();
+        });
+    });
+
+    it("POST /Discussions Bad Request", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("POST /Discussions Bad Request", (done) => {
```